_index.md

```diff
@@ -103,6 +104,11 @@ Java and C# APIs with all NIST candidate support, and some older ones. [Availabl
 
 The [Bouncy Castle for kotlin](https://github.com/bcgit/bc-kotlin) open source package provides a script/command line interface for generating certificate chains with different algorithms, including hybrid certificates using X.509 section 9.8 alternative signatures. 
 
+## Nexus Group (IN Groupe)
+PQC online test service available starting April 2024.
+Nexus Group secures society by enabling trusted identities for people and things. Our future-proof PKI helps organizations enable trust with certificate-based identities for citizens, employees, devices, applications, things, and IoT. Our PKI, Certificate Manager is certified Common Criteria (CC) EAL4+ and we deliver it on Premise and as Service.
```

Comment:
I think this text is too salezy, the list should be pure fact based. Look at the other vendor sections for guidance to keep it to the bare facts of the features offered by product itself. The company itself needs no presentation.

Reply:
OK I understand, I made an additional commit to propose an updated version.